### PR TITLE
Add Mayhem support

### PR DESCRIPTION
**Pull request by WGibbs15**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+project: wgibbs15/simd-json-fuzz
```
Comment (xansec) [inline Mayhemfile:1]: Could you please make sure the project name is generic?

File: fuzz/Cargo.toml [hunk 2]
```diff
@@ -14,6 +14,7 @@ libfuzzer-sys = "0.3"
```
Comment (xansec) [inline fuzz/Cargo.toml:14]: Can you describe the need for this change?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzz/Cargo.toml (hunk 2 above)
Comment (WGibbs15) [reply fuzz/Cargo.toml:14]: libfuzzer-sys has a current version 0.4, there's no need to use an older version

Comment (xansec) [reply fuzz/Cargo.toml:14]: Thanks!

**Pull request merged**